### PR TITLE
Handle Invalid User error coming from madmin

### PR DESCRIPTION
**Pull request by dvaldivia**
Right now we display `Internal Server Error` when invalid credentials are presneted, this makes it so we only present `Unauthorized`

### CODE REVIEW

**Review by cesnietor — COMMENTED** [review]
If minio changes the message this will no longer happen, I suggest using ErrAdminNoSuchUser from minio.

**Comment by dvaldivia** [comment]
@cesnietor i tried but that error is an integer so it's hard to compare against as it gets transformed into a api response which makes the comparison harder

File: restapi/error.go [hunk 1]
```diff
@@ -86,6 +87,11 @@ func prepareError(err ...error) *models.Error {
 			errorCode = 401
 			errorMessage = errorGenericInvalidSession.Error()
 		}
+		// console invalid session error
+		if strings.Contains(err[0].Error(), "The specified user does not exist.") {
+			errorCode = 401
+			errorMessage = errorGenericInvalidSession.Error()
+		}
 		// if we received a second error take that as friendly message but dont override the code
```
Comment (harshavardhana) [inline restapi/error.go:95]: you can use `madmin.ToErrorResponse(err).Code == "XMinioAdminNoSuchUser"`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: restapi/error.go (hunk 1 above)
Comment (dvaldivia) [reply restapi/error.go]: done!

**Review by bexsoft — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**